src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Options.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Notifier\Bridge\Lox24;
+
+use DateTimeInterface;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+final class Lox24Options implements MessageOptionsInterface
+{
+    public function __construct(
+        private array $options = [],
+    ) {
+    }
+
+    public function toArray(): array
+    {
+        return $this->options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return null;
+    }
+
+
+    /**
+     * DateTime object of SMS the delivery time.
+     * If Null or not set, the message will be sent immediately.
+     * E.g. deliveryAt(new DateTime('2024-03-21 12:17:00')) or deliveryAt(null)
+     */
+    public function deliveryAt(?DateTimeInterface $deliveryAt = null): self
```

Comment:
I suggest removing the `null` default value. Calling the method without argument would be confusing IMO.

Reply:
@stof null is 'sent ASAP'. How to provide that setting without null?

Reply:
I asked to remove the default value, not the nullability.

Reply:
done